code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -47,6 +47,79 @@
 	close_sound = 'sound/blank.ogg'
 	open_sound_volume = 25
 	close_sound_volume = 50
+	var/sealed = FALSE // is the casket sealed? If not, we can still open and close it freely.
+	var/consecrated = FALSE // Is the casket consecrated (AKA was there someone inside when we sealed it)?
+
+/obj/structure/closet/crate/coffin/open(mob/living/user)
+	if (!sealed) // if it's not sealed, process as usual.
+		. = ..()
+	else // if it's sealed, you must unseal it with a sharp object first.
+		to_chat(user, span_warning("[src] is sealed with red tallow, you must slice it open with a dagger or knife first."))
```

Comment:
Change the attack hand interaction instead, open is not just called by mobs clicking on it